study.md

```diff
@@ -29,21 +29,25 @@ Asynchronous - operations that may start in a certain sequence and may end in an
 In your own words, describe the event loop and how it works.
 
 ```md
-<!-- your answer here -->
+The event loop is a holding place for callback functions that have been returned by their enclosing function, but are waiting to execute on the stack. The stack must clear before the the callback is executed.
```

Comment:
Close, the queue is actually where the callback would wait. The event loop is what moves it to the stack once the stack is empty.